solr/core/src/java/org/apache/solr/cloud/Overseer.java

```diff
@@ -92,7 +92,7 @@
   // System properties are used in tests to make them run fast
   public static final int STATE_UPDATE_DELAY = ZkStateReader.STATE_UPDATE_DELAY;
   public static final int STATE_UPDATE_BATCH_SIZE = Integer.getInteger("solr.OverseerStateUpdateBatchSize", 10000);
-  public static final int STATE_UPDATE_MAX_QUEUE = 20000;
+  public static final int STATE_UPDATE_MAX_QUEUE = Integer.getInteger("solr.OverseerStateUpdateMaxQueueSize", 200000);;
```

Comment:
We have large cluster of 256 nodes. In case of failure, we have seen that node unable to restart because of this limit. by looking code I don't see any issue with it. But, suggest appropriate!

Reply:
Should we make the queue size change in a separate PR?

Reply:
Keeping system property for now, that we can update!

Reply:
Sounds good! Though can we make that code change in an isolated PR?